sdks/python/apache_beam/testing/benchmarks/nexmark/queries/nexmark_query_util.py

```diff
@@ -0,0 +1,36 @@
+#
```

Comment:
why not put the helpers in nexmark_util? one util file should be good enough to hold all related helper methods/classes.

Reply:
My thinking was `nexmark_query_util.py` would only contain methods that are used in queries but certainly we can change them and put them into `nexmark_util.py`